src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -332,7 +345,8 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, fromAddress, onNe
           await queryClient
             .fetchQuery({
               queryKey: thorchainSaversWithdrawQuoteQueryKey,
-              queryFn: getThorchainSaversWithdrawQuoteQueryFn,
+              queryFn: () =>
+                fetchThorchainWithdrawQuote({ asset, accountId, amountCryptoBaseUnit }),
```

Comment:
Note, with the takeaways from react-query type-safety, we're not using a magic `queryFn` which leverages the queryKey, but a good ol' async fn which is type-safe